test/jdk/java/awt/event/ComponentEvent/ListItemEventsTest.java

```diff
@@ -77,19 +77,17 @@ public static void initializeGUI() {
     public static void main(String[] s) throws Exception {
         robot = new Robot();
         try {
-
-            robot.setAutoDelay(100);
+            robot.setAutoDelay(MOUSE_DELAY);
             robot.setAutoWaitForIdle(true);
 
             EventQueue.invokeLater(ListItemEventsTest::initializeGUI);
             robot.waitForIdle();
 
             Point listAt = list.getLocationOnScreen();
-
-            Dimension listDimension = list.getSize();
-
-            robot.mouseMove(listAt.x + listDimension.width / 2,
-                listAt.y + listDimension.height / 2);
+            Dimension listSize  = list.getSize();
+            
+            robot.mouseMove(listAt.x + listSize .width / 2,
+                listAt.y + listSize .height / 2);
```

Comment:
No space before the dot:
```suggestion
            robot.mouseMove(listAt.x + listSize.width / 2,
                listAt.y + listSize.height / 2);
```